recipes/allegro5/all/conanfile.py

```diff
@@ -0,0 +1,313 @@
+from conans import ConanFile, CMake, tools
+from pathlib import Path
+import os
+
+# conan create . -e CONAN_SYSREQUIRES_MODE=enabled
+
+class Allegro5Conan(ConanFile):
+    name = "allegro5"
+    version = "5.2.7"
+    license = "Custom licenses"
```

Comment:
please, list the license names correctly: https://github.com/liballeg/allegro5/blob/master/LICENSE.txt